plugins/woocommerce-admin/client/hooks/useIsScrolled.js

```diff
@@ -3,12 +3,18 @@
  */
 import { useEffect, useRef, useState } from '@wordpress/element';
 
+function isAtBottom() {
+	return window.innerHeight + window.scrollY >= document.body.scrollHeight;
+}
+
 export default function useIsScrolled() {
```

Comment:
These changes where necessary to know if the user scrolled to the bottom and hide the border within the variation switcher.